### PR TITLE
[Docs] Update Nighthawk Description

### PR DESCRIPTION
**Pull request by theBeginner86**
Signed-off-by: Pranav Singh <pranavsingh02@hotmail.com>

**Description**
This PR updates the NightHawk description.

This PR fixes #

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by vedant-kakde — APPROVED** [review]
LGTM 👍🏻

**Review by warunicorn19 — APPROVED** [review]
LGTM. Thanks @theBeginner86

File: README.md [hunk 1]
```diff
@@ -8,11 +8,12 @@
...
 
 ## Nighthawk and Meshery
 
-Meshery integrates Nighthawk as one of (currently) three choices of load generator for characterizing and managing the performance of service meshes and their workloads. 
+Meshery integrates Nighthawk as one of (currently) three choices of load generator for characterizing and managing the performance of service meshes and their workloads.
+Centric to the advancement of Nighthawk is the Meshery and Service Mesh Performance projects, which enable Nighthawk’s standards-based distributed performance management. The intersection of these projects allow researchers and users to conveniently identify the optimal service mesh configuration while considering their specific environment, application and load. Meshery orchestrates multiple instances of Nighthawk (horizontal scaling) and provides an easy to use interface for Nighthawk’s adaptive load controller capability. 
 
 ## Why GetNighthawk?
```
Comment (leecalcote) [inline README.md:18]: Reference to "GetNighthawk" needs to be removed.

Comment (theBeginner86) [reply README.md:18]: I've fixed this at: https://github.com/layer5io/getnighthawk/pull/209

**Pull request merged**